easybuild/easyconfigs/b/bitarray/bitarray-1.2.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'bitarray'
+version = '1.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ilanschnell/bitarray'
+description = "bitarray provides an object type which efficiently represents an array of booleans"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['050cd30b810ddb3aa941e7ddfbe0d8065e793012d0a88cb5739ec23624b9895e']
```

Comment:
@stevecdavis could you look again at the `checksum`, please?

Reply:
Sorry, not sure why the checksum was wrong.  I updated it and re-submitted it.